src/validation/confirm.to.remove.validation.ts

```diff
@@ -2,9 +2,9 @@ import { body } from "express-validator";
 import { DoYouWantToRemoveKey } from "../model/data.types.model";
 
 export const confirmToRemove = [
-  body(DoYouWantToRemoveKey).custom((value, { req }) => {
+  body(DoYouWantToRemoveKey).custom((value) => {
     if (value === undefined) {
-      throw new Error("Are you sure you want to remove " + req.body['boMoName'] + "?");
+      throw new Error("Are you sure you want to remove confirm has no value");
```

Comment:
this error message doesn't really make sense to me .. has it been run past UI/Product teams?

Reply:
I don't think we run service errors past product. How about "Are you sure you want to remove confirmation has no beneficial owner or managing officer"

Reply:
The validation middleware uses that message for the GOVUK error summary it displays to the user though, for example I just tried here:

<img width="300" alt="Screenshot 2023-11-13 at 14 13 36" src="https://github.com/companieshouse/overseas-entities-web/assets/117734784/ea0ca014-8cbc-4f87-b835-f508e9be6284">

So it would have to be a product decision to change that

Reply:
Added to ticket.

Reply:
asked Mathew to review strings. But ticket unlikely to be pulled in now :-(

Reply:
And pushed change to "Are you sure you want to remove beneficial owner or managing officer"